nncf/torch/automl/environment/quantization_env.py

```diff
@@ -281,8 +287,21 @@ def _create_quantizer_table(self) -> pd.DataFrame:
         assert len(set(self.qconfig_space_map.keys()) - set(adjq_gid_map.keys())) == 0, \
             "both qconfig_space_map and adjq_gid_map must have exact keys."
 
+        # AutoQ requires quantizers in execution order
```

Comment:
Does it really "require" quantizers to be handled in a specific order, or is it merely that the reference graphs up to now have been built using the execution order of quantizers?

Reply:
@vshampor yes, it is by design to conform the Markov property of RL. It could possibly work without the order but given that we have verified the convergence in the current design, we should preserve the dynamics. Does it make sense?

Reply:
Please add this information to the comment and I'll merge this.

Reply:
@vshampor I've rebased and added the comment.